README.md

```diff
@@ -1,122 +1,129 @@
-# eap-microprofile-test-suite
- a small standalone test suite for MicroProfile on WF/EAP
+# EAP Microprofile test suite
+ 
+A small standalone test suite for MicroProfile on WildFly/EAP.
+
+## Supported MicroProfile version
+
+Tests have been executed against WildFly and EAP XP versions supporting 
+[MicroProfile 4.1 specs](https://projects.eclipse.org/projects/technology.microprofile/releases/4.1) 
+
+## Supported MicroProfile specs
+
+Currently, the following MicroProfile specs are supported:
+
+* [MicroProfile Config](https://github.com/eclipse/microprofile-config)
+* [MicroProfile JWT RBAC](https://github.com/eclipse/microprofile-jwt-auth)
+* [MicroProfile Health](https://github.com/eclipse/microprofile-health)
+* [MicroProfile Fault Tolerance](https://github.com/eclipse/microprofile-fault-tolerance)
+* [MicroProfile Metrics](https://github.com/eclipse/microprofile-metrics)
+* [MicroProfile OpenAPI](https://github.com/eclipse/microprofile-open-api)
+* [MicroProfile OpenTracing](https://github.com/eclipse/microprofile-opentracing)
+
+More tests could be added, specifically covering the MicroProfile 
```

Comment:
This is obviously true, always more tests could be added. But if we need to track this in upstream, I prefer to use github issue https://github.com/jboss-eap-qe/eap-microprofile-test-suite/issues with the description of the details.

Reply:
+1, removing.